dts/bindings/gpio/nxp,pcf857x.yaml

```diff
@@ -1,16 +1,20 @@
 # Copyright (c) 2022 ithinx GmbH
+# 2023 Amrith Venkat Kesavamoorthi <amrith@mr-beam.org>
+# 2023 Mr Beam Lasers GmbH.
 # SPDX-License-Identifier: Apache-2.0
 
-description: PCF8574 8-bit I2C-based I/O expander
+description: PCF857x 8/16-bit I2C-based I/O expander
 
-compatible: "nxp,pcf8574"
+compatible: "nxp,pcf857x"
```

Comment:
There seems to be at least one board in-tree that would need its compatible updated.
Would also require an entry in the Migration guide for 3.6 as this renaming will break users (or the old binding could be kept and deprecated?)

Reply:
I updated the release-notes-3.6.rst. Also when checking the boards in tree for the compatibles I found :
* `compatible = "nxp,pcf8523";`
* `compatible = "nxp,pcf85063a";`
The other usage was in `tests/drivers/build_all/gpio/app.overlay` where the usage is `compatible = "nxp,pcf8574";` which has already been updated with the previous commit.
Is there something I am missing? Please feel free to comment on the release notes if it needs changes. :)

Reply:
I am referring to this board :) https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/xtensa/kincony_kc868_a32/kincony_kc868_a32.dts

Reply:
Just realized it's fairly new so maybe you don't have it in your branch ;)

Reply:
Ah i see. i'll update my branch